src/browser/index.jsx

```diff
@@ -395,6 +401,12 @@ var MattermostView = React.createClass({
       }
     });
 
+    webview.addEventListener('page-title-updated', function(event) {
```

Comment:
Sure, so we should not handle `*` here. I think the point is that this event is fired even if the tab is not activated. So the listener should check whether its webview is actually shown.

Or, you might be able to add/remove the listener in `handleSelect`, but probably the code would be complex.